README.asciidoc

```diff
@@ -84,17 +75,17 @@ A number of default values are used when installing Ansible Tower on the target
 * `tower_pg_password`: The password required to login to PostgreSQL (default password is `CHANGEME`)
 * `tower_openshift_pg_pvc_size`: Size of Postgres persistent volume. Defaults to `100Gi` which is recommended for production environments
 
-### 3.1 Tower Host Name Update
+=== Tower Host Name Update
 
 Once the new Tower instance has successfully been installed on the target Openshift cluster, the host name of the new Tower instance must be placed into the `tower_host` variable value, which is located in the `<ENVIRONMENT>_tower_credentials_list.yml` file in the `external_credentials_repo` project.
 
 ```bash
 <env>_tower_host: 'tower.example.com'
```

Comment:
Wondering should this not also include:
```
pds_tower_verify_ssl
pds_tower_username
pds_tower_password
pds_tower_license
```
Not sure supplying the tower host is enough for the bootstrap process to work.